scipy/linalg/tests/test_basic.py

```diff
@@ -928,6 +928,10 @@ def test_check_finite(self):
         a_inv = inv(a, check_finite=False)
         assert_array_almost_equal(dot(a, a_inv), [[1, 0], [0, 1]])
 
+    def test_empty(self):
+        a = np.array([]).reshape((0,0))
+        a_empty = inv(a)
+        assert_array_almost_equal(a_empty, a)
```

Comment:
Just a minor comment for now--see testing guideline here re: `assert_allclose`

https://scipy.github.io/devdocs/dev/missing-bits.html#test-functions-from-numpy-testing

Reply:
Updated the tests, thanks for the help.